frontend/src/components/applyForm/ApplyForm.tsx

```diff
@@ -63,56 +68,44 @@ const ApplyForm = ({
     );
   }
 
-  let fields: JSX.Element[] = [];
-  try {
-    fields = buildFormTreeRecursive({
-      errors: saved ? validationWarnings : null,
-      formData: formObject,
-      schema: formSchema,
-      uiSchema,
-    });
-  } catch (e) {
-    console.error(e);
-    return (
-      <Alert data-testid="alert" type="error" heading="Error" headingLevel="h4">
-        Error rendering form
-      </Alert>
-    );
-  }
-
   return (
-    <>
-      <form
-        className="flex-1 margin-top-2 simpler-apply-form"
-        action={formAction}
-        // turns off html5 validation so all error displays are consistent
-        noValidate
-      >
-        <div className="display-flex flex-justify">
-          <div>{required}</div>
-          <Button
-            data-testid="apply-form-save"
-            type="submit"
-            name="apply-form-button"
-            className="margin-top-0"
-            value="save"
-          >
-            {pending ? "Saving..." : "Save"}
-          </Button>
-        </div>
-        <div className="usa-in-page-nav-container">
-          <FormGroup className="order-2 width-full">
-            <ApplyFormMessage
-              saved={saved}
-              error={error}
-              validationWarnings={validationWarnings}
+    <form
+      className="flex-1 margin-top-2 simpler-apply-form"
+      action={formAction}
+      // turns off html5 validation so all error displays are consistent
+      noValidate
+    >
+      <div className="display-flex flex-justify">
+        <div>{required}</div>
+        <Button
+          data-testid="apply-form-save"
+          type="submit"
+          name="apply-form-button"
+          className="margin-top-0"
+          value="save"
+        >
+          {pending ? "Saving..." : "Save"}
+        </Button>
+      </div>
+      <div className="usa-in-page-nav-container">
+        <FormGroup className="order-2 width-full">
+          <ApplyFormMessage
+            saved={saved}
+            error={error}
+            validationWarnings={validationWarnings}
+          />
+          <AttachmentsProvider value={attachments ?? []}>
```

Comment:
removed the unnecessary fragment wrapper, thus the big diff. The real change is here